reemgawad/theatr
# Video essay activity

**What changed:**

- Added the Video Essay activity to the seeds
- Created Video Essay partial with an input that will display the video link pasted


**How to test:**
- Seed
- Login as teacher and add Video Essay activity to classroom
- Login as student and paste youtube video link in the input field -> should display the video when you save answer
- Go back into the activity as a Teacher, should see the video & be able to leave a comment
